### PR TITLE
refactor(core): Port cache config (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
Follow-up to: #10268

### CODE REVIEW

File: packages/cli/test/unit/services/cache.service.test.ts [hunk 1]
```diff
@@ -13,10 +15,13 @@ jest.mock('ioredis', () => {
...
 	describe(backend, () => {
 		let cacheService: CacheService;
+		let globalConfig: GlobalConfig;
 
 		beforeAll(async () => {
-			config.set('cache.backend', backend);
-			cacheService = new CacheService();
+			globalConfig = Container.get(GlobalConfig);
+			// @ts-expect-error globalConfig is readonly
+			globalConfig.cache.backend = backend;
```
[the remark this answers is not in the document]
Comment (ivov) [reply packages/cli/test/unit/services/cache.service.test.ts:23]: A nice side effect also is that we don't have to force-broaden the type that was being constrained by `readonly`.

**Review by netroy — APPROVED** [review]
Thanks 💟

**Comment by janober** [comment]
Got released with [`n8n@1.54.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.54.0)

File: packages/@n8n/config/src/configs/cache.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	/** Time to live (in milliseconds) for data cached in memory. */
+	@Env('N8N_CACHE_MEMORY_TTL')
+	ttl = 3600 * 1000; // 1 hour
+}
+
+@Config
+class RedisConfig {
+	/** Prefix for cache keys in Redis. */
+	@Env('N8N_CACHE_REDIS_KEY_PREFIX')
+	prefix = 'redis';
```
Comment (netroy) [inline packages/@n8n/config/src/configs/cache.ts:18]: this used to be `cache` https://github.com/n8n-io/n8n/pull/10286/files#diff-ff0f0df9be6611837d92c12ddedac7abf37d0ede7164509c299066a8871837e0L688

**Pull request merged**